include/rocksdb/c.h

```diff
@@ -1114,9 +1114,15 @@ rocksdb_sstfilewriter_create_with_comparator(
     const rocksdb_comparator_t* comparator);
 extern ROCKSDB_LIBRARY_API void rocksdb_sstfilewriter_open(
     rocksdb_sstfilewriter_t* writer, const char* name, char** errptr);
-extern ROCKSDB_LIBRARY_API void rocksdb_sstfilewriter_add(
```

Comment:
I cannot keep it in c.h because when compile:
db/c.cc: In function ‘void rocksdb_sstfilewriter_add(rocksdb_sstfilewriter_t*, const char*, size_t, const char*, size_t, char**)’:
db/c.cc:2855:76: error: ‘rocksdb::Status rocksdb::SstFileWriter::Add(const rocksdb::Slice&, const rocksdb::Slice&)’ is deprecated [-Werror=deprecated-declarations]
   SaveError(errptr, writer->rep->Add(Slice(key, keylen), Slice(val, vallen)));